src/main/resources/plugin.yml

```
@@ -1,7 +1,8 @@
-name: Miners torch
+name: Minerstorch
```

```suggestion
name: MinersTorch
```
Think capitalizing the T would be best, not super important but yeah

---

The change is not necessary. If the author wishes to update the name to MinersTorch, they may do so post-merge or commit to my downstream branch and resolve it instead